test/lib/bamboo/adapters/ses_adapter_test.exs

```diff
@@ -106,6 +106,15 @@ defmodule Bamboo.SesAdapterTest do
     |> SesAdapter.deliver(%{})
   end
 
+  test "set custom headers on attachments" do
```

Comment:
This test example tests `mail`, `bamboo` methods but not the adapter method.

Reply:
Hi @kalys! We only need to assert the result to guarantee that the values are being passed through the underlining mail functions. We haven't changed any adapter functions worth testing since the `content_id` value is mapped directly from the bamboo message. What would be your suggestion?

Ping @rafaeliga.